src/solver/conv_hip_implicit_gemm_bwd_data_v4r1_xdlops.cpp

```diff
@@ -392,6 +392,8 @@ bool PerformanceImplicitGemmBwdDataV4R1Xdlops::IsReallyValid(const ConvolutionCo
 bool PerformanceImplicitGemmBwdDataV4R1Xdlops::IsFastToBeUsedForTuning(
     const ConvolutionContext& ctx) const
 {
+    if(use_spare_set)
+        return true;
```

Comment:
See https://github.com/ROCmSoftwarePlatform/MIOpen/pull/605#discussion_r527178430

Reply:
let @zjing14 and we discuss it

Reply:
[Recommendation] See https://github.com/ROCmSoftwarePlatform/MIOpen/pull/605#discussion_r528903383.

Resolved.